### ISSUE TITLE
lapsed member returning

### ISSUE THREAD

**Issue by jimr1603** [ppuk/ppuk-drupal-theme#14]
Not sure if this is a bug with the theme or something else.

My profile (jimr1603) shows me as a member for 6yr 10 month, when I only re-joined yesterday! 

How the site handles lapsed/returning members might need handling as an edge case. It's 'true' that I first became a member 6+ years ago, it's 'true' that I've 5+ years of membership, and it's 'true' that I've been a continuous member for 24 h.

**Comment by strideynet** [comment]
Yeah, there's a lot of issues with the drupal site. We've got plans to completely rewrite the membership system as a standalone platform in the summer, since basically nothing seems to offer the feature sets we need.

**Comment by ajehals** [comment]
Do you guys have a requirement set somewhere? There was one done before this build, complete with a decent review of what was available at the time that might be useful to compare. IIRC the membership system was reasonably flexible (That was sort of the point..) across... well everything, by role and membership type (Although that was several years ago and I don't know what's changed).

**Comment by zapto2600** [comment]
Hi Andy,

Security.... Drupal has some major flaws, also the API is no longer supported by PayPal. 

⁣Thanks
Matt

**Comment by strideynet** [comment]
Still looking to spec it out somewhat soon, so if you still have the old spec somewhere that'd be awesome @ajehals

**Comment by ajehals** [comment]
I almost certainly don't have the old spec. It'd have been in the the Campaigns or NEC NWP folder structure somewhere though.. Possibly in email (the individual department requirements almost certainly..). But that'd have been 4+ years ago.
